src/index.js

```diff
@@ -16,25 +16,29 @@ const opts = {}
 opts.jwtFromRequest = ExtractJwt.fromAuthHeaderAsBearerToken()
 opts.secretOrKey = process.env.JWT_SECRET || 'secret'
 
-passport.use(new JwtStrategy(opts, function (jwtPayload, done) {
-  models.user.findOne({ id: jwtPayload.sub }, function (err, user) {
-    if (err) {
-      return done(err, false)
-    }
+passport.use(new JwtStrategy(opts,async function (jwtPayload, done) {
+  const user = await models.user.findOne({where: { id: jwtPayload.sub }}) 
```

Comment:
Missing spaces here. Did you run the linter?
```suggestion
  const user = await models.user.findOne({ where: { id: jwtPayload.sub } }) 
```